docs/docsite/rst/community/how_can_I_help.rst

```diff
@@ -61,6 +61,13 @@ Review and submit pull requests
 
 As you become more familiar with how Ansible works, you may be able to fix issues or develop new features yourself. If you think you have a fix for a bug in Ansible, or if you have a new feature that you would like to share with millions of Ansible users, read all about the :ref:`development process <community_development_process>` to learn how to get your code accepted into Ansible.
 
+You can also get started with solving GitHub issues labeled with the ``easyfix`` and ``good_first_issue`` labels for:
+
+- `Ansible collections <https://github.com/search?q=user%3Aansible-collections+label%3Aeasyfix%2C%22good+first+issue%22+state%3Aopen&type=Issues>`_
+- `All other Ansible projects <https://github.com/search?q=user%3Aansible+user%3Aansible-community+label%3Aeasyfix%2C%22good+first+issue%22+state%3Aopen&type=Issues>`_
+
+Add a comment on the GitHub issue you chose to say you are looking at it and to ask for help if you need it.
```

Comment:
```suggestion
When you choose an issue to work on, add a comment directly on the GitHub issue to say you are looking at it and let others know to avoid conflicting work. 
You can also ask for help in a comment if you need it.
```